src/web/crate_details.rs

```diff
@@ -100,7 +102,9 @@ impl CrateDetails {
                 releases.documentation_url,
                 releases.default_target,
                 doc_coverage.total_items,
-                doc_coverage.documented_items
+                doc_coverage.documented_items,
+                COALESCE(doc_coverage.total_items_needing_examples, 0) AS total_items_needing_examples,
+                COALESCE(doc_coverage.items_with_examples, 0) AS items_with_examples
```

Comment:
Hmm, I don't think this should be a COALESCE. If there's no data, we shouldn't display the percentage, not say that it's 0.

Reply:
Ok!